Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -239,7 +239,7 @@ def test_python_step(self):
         # be compatible
         frame_name = pyframe.co_name.proxyval(set())
         self.assertEqual(frame_name, 'join')
-        assert re.match(r'\d+    def join\(', result), result
+        self.assertRegexpMatches(result, r'^\d+    def join\($'), result
```

Comment:
Appending `$` changes the regex.

Reply:
Yeah, sorry, forgot that the match only compares to the beginning

Reply:
[resolved](https://github.com/cython/cython/pull/4340/commits/7cd4748b7e9422d2a630f6ef902127516c8a0030)